README.md

```diff
@@ -258,6 +258,7 @@ $rabbitmq_defaults = [
                 'ssl_context' => null,
                 'keepalive' => false,
                 'heartbeat' => 0,
+                'channel_rpc_timeout' => 1
```

Comment:
Why it is 1 here, as it's actually 0.0 in the configuration defaults?

Reply:
correct, updated